deployment/terraform/outputs.tf

```diff
@@ -1,3 +1,3 @@
-output "service_elb_ip" {
-  value = "${aws_elb.gtsite.dns_name}"
+output "service_alb_dns_name" {
```

Comment:
I'm not sure; it's a holdover from the old setup. I didn't want to delete it in case one of the other services needed it.